content/blog/2025-03-20-new-hash-table.md

```diff
@@ -0,0 +1,202 @@
++++
+title= "A new hash table"
+date= 2025-03-20 00:00:00
+description= "Designing a state-of-the art hash table implementation"
+authors= [ "zuiderkwast", "SoftlyRaining"]
++++
+
+Valkey is essentially a giant hash table attached to the network. A hash table
```

Comment:
Probably want a better hook. We can talk about how many caching workloads are bound on storing data, so being able to to store more data allows you to reduce the size of your clusters.